src/components/ContentWidgets/ImagesInRow/ImagesInRow.tsx

```diff
@@ -16,24 +16,23 @@ type linkCaptionPair = {
  * July 14, 2019
  */
 export const ImagesInRow: React.FC<ContentSingularData> = ({
-    imagesInRow_linkCaptionPairs
+    imagesInRow_items
 }) => {
-    if (!imagesInRow_linkCaptionPairs) {
+    if (!imagesInRow_items) {
         return <></>;
     }
     return <>
         <div className="img-row">
-            {imagesInRow_linkCaptionPairs.map((pair: linkCaptionPair, index: number) => (
+            {imagesInRow_items.map((pair: imgRowItem, index: number) => (
                 <div className="img-row-item">
                     <figure>
                         <a href={pair.extLink ? pair.extLink : pair.imgLink}>
```

Comment:
Last year I had it so the image _and_ caption were the link. I would suggest doing that